include/swift/SymbolGraphGen/SymbolGraphOptions.h

```diff
@@ -35,6 +35,10 @@ struct SymbolGraphOptions {
   /// Emit members gotten through class inheritance or protocol default
   /// implementations with compound, "SYNTHESIZED" USRs.
   bool EmitSynthesizedMembers;
+  
+  /// Whether to silence the "Found N symbols" messages when rendering
```

Comment:
It's more than the "Found N symbols" messages though right?

Reply:
There are three lines that are affected by this change:

```
Emitting symbol graph file for module: <filename>.swiftmodule
X top-level declarations in this file.
Found Y symbols and Z relationships.
```

`emitSymbolGraphForModule` only emits the latter two (the first is printed directly by `swift-symbolgraph-extract`), so centering the docs around the "Found N symbols" seemed like the most direct way to describe it. I could mention the "X top-level symbols" line as well if that seems better.

Reply:
That's true today but might not be in the future. I'm nitpicking though, this isn't blocking :)